libcxx/test/std/ranges/range.adaptors/range.zip.transform/cpo.pass.cpp

```diff
@@ -0,0 +1,159 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+
+// std::views::zip_transform
```

Comment:
I think we're missing some tests for the "function object" aspect of this CPO. For example, if `views::zip_transform` were not copy-constructible, this test would pass but you wouldn't be able to pass it to a higher-order function. I think we're missing an addition to https://github.com/llvm/llvm-project/blob/main/libcxx/test/std/library/description/conventions/customization.point.object/cpo.compile.pass.cpp.